### PR TITLE
fix: removed workaround for connectors in OpenShift routes

### PR DESCRIPTION
**Pull request by hamza-m-masood**
## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/self-managed/setup/deploy/openshift/redhat-openshift.md [hunk 1]
```diff
@@ -326,37 +326,6 @@ tasklist:
 The actual configuration properties can be reviewed [in the Tasklist configuration documentation](/self-managed/tasklist-deployment/tasklist-configuration.md#zeebe-broker-connection).
 
 5. For Connectors:
```
Comment (conceptualshark) [inline docs/self-managed/setup/deploy/openshift/redhat-openshift.md:328]: Can this be merged into step 4 now that Connectors don't have a separate set of requirements/steps?

**Review by conceptualshark — COMMENTED** [review]
Only one question on if steps 4 and 5 can be condensed. Otherwise happy for this to be merged once it's good to go from distribution.

**Comment by conceptualshark** [comment]
@hamza-m-masood Is this needed for the alpha release? Does it still need distribution review?

**Review by aabouzaid — DISMISSED** [review]
LGTM, thank you :+1:

File: docs/self-managed/setup/deploy/openshift/redhat-openshift.md (hunk 1 above)
Comment (hamza-m-masood) [reply docs/self-managed/setup/deploy/openshift/redhat-openshift.md:328]: @conceptualshark Connectors have a different set of env vars. I think it is still a good idea to have them separate.

**Comment by hamza-m-masood** [comment]
@conceptualshark It is reviewed from distribution now. Feel free to merge.

**Review by conceptualshark — DISMISSED** [review]

**Comment by hamza-m-masood** [comment]
@conceptualshark can this be merged?

**Pull request merged**